src/databricks/labs/ucx/recon/migration_recon.py

```diff
@@ -145,5 +146,5 @@ def _percentage_difference(result: DataComparisonResult) -> int:
         return round(abs(source_row_count - target_row_count) / source_row_count * 100)
 
     def _try_fetch(self) -> Iterable[ReconResult]:
-        for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
+        for row in self._fetch(f"SELECT * FROM {escape_sql_identifier(self.full_name)}"):
```

Comment:
It could be?

It's a reasonably large refactor though, and would obfuscate a bit the intent of this PR which was to cover some missed cases from the earlier PR.

(I'd probably also go with `full_name_escaped` to match `TableIdentifier` somewhat.)

@ericvergnaud: What do you think?

Reply:
I would introduce it without refactoring the entire code ?

Reply:
On balance I think right now I'd prefer consistency with the existing conventions being used elsewhere.

(To be clear I'm not against changing the convention, but as mentioned that's a much larger set of changes and I'd prefer to limit the scope of the PR.)